pkg/model/api/v1/datasource/prometheus.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package datasource
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+var defaultPrometheusAllowedEndpoints = []HTTPAllowedEndpoint{
+	{
+		Endpoint: "/api/v1/labels",
+		Method:   http.MethodPost,
```

Comment:
Hm, is this not GET?

Reply:
We support both methods:

https://github.com/prometheus/prometheus/blob/d77c985f8c122cf5947b0eb621999fcc8dcca9bf/web/api/v1/api.go#L294-L295

...I think someone in the past once ran into problems with `GET` and the URL length limit for match parameters, so for newer Prometheus versions, we support `POST` as well for some of these endpoints.

Reply:
actually both are supported. It's just POST is avoiding the size limit of the URL, because the parameters are passed in the body